app/controllers/v1/pacbio/runs/wells_controller.rb

```diff
@@ -63,22 +63,12 @@ def param_names
           well_param_names(p1)
```

Comment:
Your changes here are fine, but I think there's a bit of scope for overall improvement here. A few things I've noticed:

1) A lot of code repetition, we seem to be permitting things several times
2) We're passing round various flavours of 'params' despite it being an object variable
3) We've a few points where we're chaining hash [] methods, and then assuming we're getting a particular
    data type back. eg. `params.require(:relationships)[:libraries][:data].map`